### PR TITLE
Add container logic

### PR DESCRIPTION
**Pull request by alayek**
- [x] Add container logic to test max width of the header and content ( #6 )
- [x] Fix Nunito font not being loaded as described in #4  
- [x] Consume Next Themes package (#7 )
- [x] Mobile menu ( #8 )

---

Note that there are some design issues in mobile NAV, will fix with a future PR.

Also, it seems Nunito font is still not loading. Should be subject of another PR.

### CODE REVIEW

**Comment by crimelabs786** [comment]
@alayek please mark this as a draft PR or WIP if it's not ready to merge yet.

**Comment by alayek** [comment]
Closes #6

**Comment by financenoob** [comment]
Bit of a nitpick. You should extract long class names into variables or functions if they are to be changed using state.

**Comment by financenoob** [comment]
Next time, please scope your PR to a single concern or issue.

**Review by financenoob — CHANGES_REQUESTED** [review]

**Comment by financenoob** [comment]
> @financenoob have addressed your concerns, and moved the class logic to `useMemo`. Do you feel using memo is an overkill here?

It does seem overkill here. I would err on removing it.

File: hooks/use-dark-mode/use-dark-mode.tsx [hunk 1]
```diff
@@ -1,76 +1,17 @@
...
-    /**
-     * No value was set in localstorage
-     * Or localstorage doesn't exist
-     * Need to get from user system preference
-     */
-    if (typeof window.matchMedia === 'function') {
-      const userPrefersDarkTheme = window.matchMedia(
-        '(prefers-color-scheme: dark)',
-      );
+  const { theme, setTheme } = useTheme();
```
[the remark this answers is not in the document]
Comment (alayek) [reply hooks/use-dark-mode/use-dark-mode.tsx]: Fixed, thanks for pointing out.

**Comment by crimelabs786** [comment]
Let's merge it for now, we can address pending stuff in future PRs.

**Review by crimelabs786 — APPROVED** [review]

**Review by financenoob — APPROVED** [review]

**Pull request merged**